### PR TITLE
Fix minor bugs in UG

### PR DESCRIPTION
**Pull request by huangjiaxi1111**
* Fixed duplicated handling
* Added description to various parts
* Added chained find
* Added extra details in add command description

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -97,24 +97,22 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME (p/PHONE_NUMBER | e/EMAIL) (r/MODULECODE[-ROLETYPE])+ [a/ADDRESS] [t/TAG]+`
+Format: `add n/NAME (p/PHONE_NUMBER | e/EMAIL) [r/MODULECODE[-ROLETYPE]]+ [a/ADDRESS] [t/TAG]+ [d/DESCRIPTION]`
```
Comment (lithops-zty) [inline docs/UserGuide.md:100]: good catch!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -97,24 +97,22 @@ Format: `help`
...
 
 * `PHONE_NUMBER` is almost a free-form text field with minimal validation. Refer to the [input format section](#input-format) to find out more.
 * `MODULECODE` refers to a module code of a NUS module (e.g. CS1101S, MA1521)
 * `ROLETYPE` refers to one of the following: `student`, `ta`, `tutor`, `prof`, `professor`.
 * The `r/MODULECODE[-ROLETYPE]` parameter means that the person has the role for this module (e.g. `r/CS1101S-student` means that the person is a student of CS1101S).
 * In `r/MODULECODE[-ROLETYPE]`, `[-ROLETYPE]` is optional. In such cases, this means that the person is a student of that module (e.g `r/MA1521` means that the person is a student of MA1521).
 * If the same module is added multiple times, then it is assumed to be an error in user input, because a person should not have multiple roles (student, tutor, professor) at the same time (e.g. `r/CS1101S-student r/CS1101S-prof` is not allowed).
+* `ADDRESS` can take any values and can not be blank.
+* `TAG` can take any alphanumeric values and can not be blank.
+* `DESCRIPTION` can take any values.
```
Comment (lithops-zty) [inline docs/UserGuide.md:110]:
```suggestion
* `DESCRIPTION` can take any values but cannot exceed 500 characters.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -97,24 +97,22 @@ Format: `help`
...
 <box type="tip" seamless>
 
-**Tip:** A person must have at least one pair of module code and role type.
-</box>
-
-<box type="warning" seamless>
-
 **Duplicate Handling:**
-A person is considered a duplicate if another person in the address book has the same email address. The app will prevent adding contacts with duplicate emails. 
+A person is considered a duplicate if another person in the address book has the same email address or phone number. The app will prevent adding contacts with duplicate emails or phone numbers.
```
Comment (lithops-zty) [inline docs/UserGuide.md:115]: 👍 Do you think we should mention the 2 checks only happen when the email or phone number of the person to be added is non-empty?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -185,22 +183,19 @@ Examples:
...
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]+​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]+ [d/DESCRIPTION]`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `t/` without
   specifying any tags after it.
+* Similarly, you can remove a person's description by typing `d/` without specifying any description after it.
```
Comment (lithops-zty) [inline docs/UserGuide.md:198]: Good!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -257,7 +252,18 @@ Examples:
 * `find n/John n/Ben r/cs1101s r/ma1522` return all persons whose name are either John or ben, taking either CS1101S or MA1522
   ![result for 'find n/John n/Ben r/cs1101s r/ma1522'](images/findNameAndModuleExample.png)
 
-### Deleting a person or multiple persons: `delete`
+<box type="info" seamless>
+
+**Chained Find:**
+To search from the previously displayed results, use `find chained`.<br>
+Example: `find n/John` followed by `find chained n/Doe` will return all persons whose name contains `John` and `Doe`.
```
Comment (lithops-zty) [inline docs/UserGuide.md:259]:
```suggestion
To further filter the previous search results, use `find chained`.<br>
Example: `find n/John` followed by `find chained n/Doe` will return all persons whose name contains `John` and `Doe`.
```

**Review by lithops-zty — CHANGES_REQUESTED** [review]
Good! Some small errors to be fixed and suggestions you can consider

File: docs/UserGuide.md (hunk 3 above)
Comment (huangjiaxi1111) [reply docs/UserGuide.md:115]: I think should be fine, it is quite intuitive

**Review by lithops-zty — APPROVED** [review]
👍

**Pull request merged**